ts/generators/app/providers/hippo.ts

```diff
@@ -34,6 +34,12 @@ export const hippo: Registry = {
               message: "What storage ID (bindle name) would you like for your Hippo app?",
               default: bindleise(ans.authorName, ans.moduleName),
             },
+            {
+              type: 'input',
+              name: 'domainName',
+              message: "What domain name would you like for your Hippo app?",
+              default: process.env['APP_URL'] || ans.moduleName + '.hippos.rocks',
```

Comment:
I'm not sure there's any value in taking an EV here.  This isn't something that would be already set up for repeat use (whereas the Bindle and Hippo URLs and auth will be used persistently).  So setting up an EV is as expensive as just typing it in.

Reply:
okay! I figured this would be a quibble. Sounds good to me :)